corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,7 +1,7 @@
 from corehq.util.elastic import es_index
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-DOMAIN_INDEX = es_index("hqdomains_20160531_1131")
+DOMAIN_INDEX = es_index("hqdomains_20160622_0645")
```

Comment:
fyi, this broke the build (we have tests that explicitly check the index and checkpoint names to avoid accidental index changes and checkpoint rewinds).